pkg/cmd/task/start.go

```diff
@@ -132,6 +134,7 @@ like cat,foo,bar
 	c.Flags().StringSliceVarP(&opt.Labels, "labels", "l", []string{}, "pass labels as label=value.")
 	c.Flags().BoolVarP(&opt.ShowLog, "showlog", "", true, "show logs right after starting the task")
 	c.Flags().StringVarP(&opt.Filename, "filename", "f", "", "filename containing a task definition")
+	c.Flags().Int64VarP(&opt.TimeOut, "timeout", "t", 0, "timeout for taskrun in seconds")
```

Comment:
I think the default timeout is 60 mins. [Link](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md#configuring-default-timeout) for reference

Reply:
Yeah, misread the 0 as default. Thanks!